headless-cms/admin/components/ContentField.tsx

```diff
@@ -21,28 +25,53 @@ const inlineButtons: BlockEditorProps["inlineButtons"] = [
 		RB.headingOne,
 		RB.headingTwo,
 		RB.headingThree,
-		RB.headingFour,
+		RB.headingFour
```

Comment:
🤡

Reply:
It's correct in second PR. It'll be right after rebase.